src/sweeps/config/schema.py

```diff
@@ -86,6 +86,10 @@ def fill_parameter(parameter_name: str, config: Dict) -> Optional[Tuple[str, Dic
         except jsonschema.ValidationError:
             continue
         else:
+            if schema_name == "param_dict":
+                # Don't try and fill default values for these
+                # just pass the config forward.
+                return schema_name, config
```

Comment:
can we just raise a custom exception here for type = "param_dict" and then handle it on line 385 of _unnest. that way we dont have to implement stubs TYPE=param_dict in HyperParemeter

Reply:
thats a pretty good idea, makes for less lines of code too